504: change installation steps description to make it work (#215)
streamnative/function-mesh-website: docs/install-function-mesh.md

- This example shows how to verify whether Function Mesh can run properly.

    After installing the Function Mesh Operator and deploying a Pulsar cluster, you can submit a sample CRD to create Pulsar Functions, source, sink, or Function Mesh. In this example, a CRD is submitted to the Pulsar cluster for creating a Pulsar Function. You can also submit other CRDs under the `./config/samples` directory.
    

>     1. Change the CRD definition to point to your Pulsar cluster.

Reviewer: ```suggestion
    1. Update your Pulsar cluster configurations in the CRD.
```
